basic-algorithms-&-datastructures/index000.js

```diff
@@ -0,0 +1,16 @@
+const newsFeed = {
+}
+
+/*
```

Comment:
These projects should have an introduction page, so we can shorten this a bit:

/*
Line one is the empty `newsFeed` object. Objects have properties which can be set to values. Properties are made up of key : value pairs: 

```js
const  objectExample = { cats: 4 } 
```

Create a property for `newsfeed` with the key of `page` and set it to the value of `'front_page'`.
*/